src/main/scala/org/ergoplatform/nodeView/wallet/persistence/OffChainRegistry.scala

```diff
@@ -90,12 +91,12 @@ case class OffChainRegistry(height: Int,
 object OffChainRegistry {
 
   def empty: OffChainRegistry =
-    OffChainRegistry(ErgoHistory.EmptyHistoryHeight, Seq.empty, Seq.empty)
+    OffChainRegistry(ErgoHistory.EmptyHistoryHeight, Array.empty, Array.empty)
```

Comment:
@jellymlg, instead of using Array.empty which allocates a new array every time, it is better to use ArraySeq.empty, which shares single immutable instance for all invocations. This will also work nicely when you rollback changes in OffChainRegistry

Reply:
done